test/e2e/node/kubelet_authz.go

```diff
@@ -73,11 +72,11 @@ func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint,
 	}, metav1.CreateOptions{})
 	framework.ExpectNoError(err)
 
-	ginkgo.By(fmt.Sprintf("Creating ClusterRole %s with for %s/%s", crName, resource, authzSubresource))
+	ginkgo.By(fmt.Sprintf("Creating ClusterRole with prefix %s with for %s/%s", authzSubresource, resource, authzSubresource))
 
-	_, err = f.ClientSet.RbacV1().ClusterRoles().Create(ctx, &rbacv1.ClusterRole{
+	clusterRole, err := f.ClientSet.RbacV1().ClusterRoles().Create(ctx, &rbacv1.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: crName,
+			GenerateName: authzSubresource + "-",
```

Comment:
Original comment from https://github.com/kubernetes/kubernetes/pull/131409#discussion_r2056567790:

Is the ClusterRole and ClusterRoleBinding ever cleaned up?

Reply:
checking

Reply:
This is a valid point. Looks like [only the namespace scoped ](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L344-L409)assets are being cleaned up.